src/lib/exec-fetcher.js

```diff
@@ -26,12 +27,13 @@ const isExe = (mode, gid, uid) => {
 }
 
 const execExist = async binPath => {
-  const binExists = await fs.exists(binPath)
-  if (!binExists) {
-    return false
+  try {
```

Comment:
`fs.exists` was not really needed here. If the file does not exist `statSync` will throw an error.

Reply:
Yep, will do it.